lib/v2/sanlian/channel.js

```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const util = require('@/v2/nintendo/utils');
+
+const rootUrl = 'https://www.lifeweek.com.cn/column';
+const articleRootUrl = 'https://www.lifeweek.com.cn/article';
+module.exports = async (ctx) => {
+    const channel = ctx.params.id;
+    const url = `${rootUrl}/${channel}`;
+    const response = await got(url, {
+        responseType: 'buffer',
+    });
+    const result = await util.nuxtReader(response.data);
+    if (!result.articleList) {
+        throw new Error('新闻信息不存在，请报告这个问题');
+    }
+    const items = await Promise.all(
+        result.articleList.map((item) => {
+            const articleLink = `${articleRootUrl}/${item.id}`;
+            return ctx.cache.tryGet(articleLink, async () => {
+                const response = await got(articleLink, {
+                    responseType: 'buffer',
+                });
+                const articleResult = await util.nuxtReader(response.data);
+                const time = timezone(parseDate(item.pubTime), +8);
+                return {
+                    title: item.title,
+                    description: articleResult.articleDetail.content,
+                    link: articleLink,
+                    pubDate: time,
+                };
+            });
+        })
+    );
```

Comment:
The content of each article can be obtained via API of `/api/article/:articleId`.